docs/installingNative.md

```diff
@@ -93,11 +95,21 @@ Most of these prerequisites can be installed using the package manager of your O
     The main prerequisites can be installed with [Homebrew](http://brew.sh/):
 
     ```sh
-    brew install git graphviz postgresql redis uv d2 node npm pgvector
+    brew install git graphviz postgresql@17 redis uv d2 node pgvector
     # Optional; needed only for some example questions that use LaTeX
     brew install texlive
     ```
 
+    You may want to start up the `postgresql` server on boot, and add binaries to your path:
+
+    ```sh
+    brew services start postgresql@17
+    # zsh
+    echo 'export PATH="$(brew --prefix postgresql@17)/bin:$PATH"' >> ~/.zshrc && . ~/.zshrc
+    # bash
+    echo 'export PATH="$(brew --prefix postgresql@17)/bin:$PATH"' >> ~/.bashrc && . ~/.bashrc
```

Comment:
This wasn't necessary for me after installing Postgres. Maybe you have conflicting versions? If you run `which psql` in a shell without this, does anything show up? For me, it points to `/opt/homebrew/bin/psql`.

Reply:
```
which psql
/opt/homebrew/opt/postgresql@17/bin/psql
```

Reply:
Removing activation means it doesn't show up for me.